engine/src/flutter/impeller/entity/geometry/round_superellipse_geometry.cc

```diff
@@ -11,6 +11,157 @@
 namespace impeller {
 
 namespace {
+
+// An interface for classes that arranges a point list that forms a convex
+// contour into a triangle strip.
```

Comment:
nit: I think we already have a class named ConvexTessellator. Though this is defined in a .cc it would probably be better to rename to somethng like SuperEllipseTessellator or something like that.

Reply:
Renamed to ConvexRearranger